src/sentry/runner/__init__.py

```diff
@@ -161,4 +165,22 @@ def call_command(name, obj=None, **kwargs):
 
 
 def main():
-    cli(prog_name=get_prog(), obj={}, max_content_width=100)
+    try:
+        if os.environ.get("SENTRY_DEVENV_DSN"):
+            logger.warning(
+                "The Sentry runner will report development issues to Sentry.io."
+                "Use SENTRY_DEVENV_NO_REPORT to avoid reporting issues."
+            )
+        cli(prog_name=get_prog(), obj={}, max_content_width=100)
+    except Exception as e:
+        if os.environ.get("SENTRY_DEVENV_DSN"):
```

Comment:
@untitaker @billyvg For me, this block is sufficient. The changes in the `sdk.py` did not work to catch issues raised and we can investigate later if we need to make changes there; works for you? (If you say so, I will go and remove all changes from `sdk.py`).

You can test this change by shutting down the devservices and running `sentry cleanup`.

You can see the issue reported here:
https://sentry.io/organizations/sentry/issues/2279751970/?project=1492057&query=is%3Aunresolved+is%3Afor_review+assigned_or_suggested%3Ame_or_none

Reply:
I've decided to remove the changes to `sdk.py`. This will be good enough for now.